cmd/server-main.go

```diff
@@ -106,8 +106,12 @@ func enableLoggers() {
 func initConfig() {
 	// Config file does not exist, we create it fresh and return upon success.
 	if isFile(getConfigFile()) {
-		fatalIf(migrateConfig(), "Config migration failed.")
-		fatalIf(loadConfig(), "Unable to load minio config file")
+		// Check if we should migrate the config,
+		// if not proceed to load the config.
+		if shouldMigrate() {
+			fatalIf(migrateConfig(), "Config migration failed.")
+		}
+		fatalIf(loadConfig(), "Unable to load config version %s", v18)
```

Comment:
Load config will fail @balamurugana for older releases, we just want to see the version not rest of the fields.